README.md

```diff
@@ -88,12 +88,23 @@ To use this bot, you will need to create a Discord bot and invite it to your ser
 
 To start the bot, run the following command:
 
-```
+```bash
 python bot.py --conf config.ini
 ```
 
 The bot will log in to Discord and start listening for messages in the configured channels. When a message is received, the bot will send the message to the OpenAI API and wait for a response. The response will be sent back to the user who sent the message.
 
+# Run in container
+
+To run the bot in a container, you will need to check out this repository, rename the `config.ini.example` to `config.ini` and fill it in appropriately.
+Then, run the following commands:
+```bash
```

Comment:
_nitpick:_ I'm not certain I care for the "bash" added here, but I'm not going to block the merge due to it.